src/tests/functional/plugin/cpu/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -76,6 +76,10 @@ INSTANTIATE_TEST_SUITE_P(
         smoke_OVClassSetModelPriorityConfigTest, OVClassSetModelPriorityConfigTest,
         ::testing::Values("MULTI", "AUTO"));
 
+INSTANTIATE_TEST_SUITE_P(
+        smoke_OVClassSetAllowAutoBatchingConfigTest, OVClassSetAllowAutoBatchingConfigTest,
+        ::testing::Values("MULTI", "AUTO"));
```

Comment:
also make sure to instantiate the tests for the GPU ("GPU", "AUTO:GPU", "MULTI:GPU", "BATCH:GPU" )in the src/tests/functional/plugin/**gpu**/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

Reply:
pls see comments on how to enable the tests for the BATCH below

Reply:
you wouldn't be able to instantiate the test for the BATCH - they will fail (as these would require the config with the device, and your current impl uses the NoConfig as a base class for the test)

Reply:
pls see https://github.com/openvinotoolkit/openvino/pull/10399/commits/13666d3ce71c8dd4888feae85a2b164c0867a284 on how to enable for the GPU and auto-batching